include/seqan/index/index_dfi.h

```diff
@@ -215,6 +215,22 @@ namespace seqan
         TPredHull        predHull;
         TPred            pred;
 
+        /*!
+         * @fn IndexDfi::Index
+         * @brief Constructor
+         *
+         * @signature Index::Index();
+         * @signature Index::Index(index);
+         * @signature Index::Index(text);
```

Comment:
Make only one signature, with optional arguments like:
Index(text [, predHull] [,pred]) and write in the param section that this is an optional parameter.